src/azul/service/responseobjects/elastic_request_builder.py

```diff
@@ -402,14 +419,14 @@ def transform_summary(
         ).bucket(
             'cell_count',
             'sum',
-            field='contents.cell_suspensions.total_estimated_cells'
+            field='contents.cell_suspensions.total_estimated_cells_ORIGINAL'
```

Comment:
I'd prefer 

```suggestion
            field='contents.cell_suspensions.total_estimated_cells_'
```

unless ES objects.